generators/devops/promptConfig.js

```diff
@@ -0,0 +1,24 @@
+const pipelineOptions = [
+    {
+        'name': 'None',
+        'value': 'none'
+    },
+    {
+        'name': 'Azure DevOps',
+        'value': 'azure'
+    }
+];
+// generat configuration options
+const configOptions = [
+    // Library selection
+    {
+        type: 'list',
```

Comment:
Choice

Reply:
AFAIK there's no type choice https://github.com/SBoudrias/Inquirer.js#prompt list seem to be doing the trick (allowing you to pick one option), were you expecting another behavior?

Reply:
The choice is called 'checkbox'. I used it when for the jquery, pnp/js and so on selection.
I will fix it.